src/colony/colony.c

```diff
@@ -57,6 +57,21 @@ int colony_isarray (lua_State* L, int index)
   return ret;
 }
 
+void colony_array_length (lua_State* L, int pos)
```

Comment:
If this isn't used anywhere, we can just get rid of it. (I expected it to be used in colony_array_length_i, but apparently it wasn't!)